### PR TITLE
Fix simple string to json conversion

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
$title.

Note: with this fix the shell print 2 pairs of double quotes.
```shell
=$ var x = "hello\"a".toJsonString();

=$ x 
""hello\"a""
```

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/28906

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/StringUtils.java [hunk 1]
```diff
@@ -374,6 +378,18 @@ public static String getJsonString(Object value) {
...
     }
 
+    private static String stringToJson(BString value) {
+        ByteArrayOutputStream byteOut = new ByteArrayOutputStream();
+        JsonGenerator gen = new JsonGenerator(byteOut);
+        try {
+            gen.writeString(value.getValue());
+            gen.flush();
+        } catch (IOException e) {
+            throw new BallerinaException("Error in converting string value to a json string: " + e.getMessage(), e);
```
Comment (rdhananjaya) [inline bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/StringUtils.java:388]: try-catch is just to handle checked exception

**Review by hasithaa — APPROVED** [review]

**Comment by hasithaa** [comment]
@pubudu91, Could you please check the shell change.

**Comment by pubudu91** [comment]
> @pubudu91, Could you please check the shell change.

Hmm. The shell seems to print string values with the quotes when printing them upon evaluation. Going by this, should probably escape the quotes of the JSON string right? e.g.,
```
"\"hello\"a\""
```

**Comment by pubudu91** [comment]
I just checked in the latest master and it doesn't seem to be printing extra quotes?
```
=$ var x = "hello\"a".toJsonString();                                                                                                                                                                                

=$ x                                                                                                                                                                                                               
"hello"a"
```

**Pull request merged**